lib/components/Masthead/Masthead.tsx

```diff
@@ -59,8 +57,14 @@ export interface MastheadUserItem {
     attr?: InlinePopup.Attributes;
 }
 
+interface MastheadAttributes {
+    brandingContainer?: DivProps;
+    branding?: SpanProps;
+    logo?: DivProps;
+}
+
 export interface MastheadProperties {
-    branding: MethodNode;
+    branding: string;

```

Comment:
any reason why?

Reply:
this changes should be reviewed in the other PR, but we were only using it as a methodNode to pass a span with the appropriate data test hook... adding attr API and making it a string will have the same effect without the extra span in the DOM